hs/src/Reach/Connector/ALGO.hs

```diff
@@ -2032,6 +2032,21 @@ ce = \case
     ca oa
     cextract start sz
     cfrombs t
+  DLE_ObjectSet at obj_a fieldName val_a -> do
+    -- This makes the (unsound?) assumption that when adding a new field to an object,
```

Comment:
This is fine, as long as the types are the same. That is, we turn a 2d point into a 3d point by doing `DLE_ObjectSet sb pointA "z" 3`